src/backend/tests/Fusion.Resources.Api.Tests/IntegrationTests/ResponsibilityMatrixTests.cs

```diff
@@ -28,8 +30,6 @@ public class ResponsibilityMatrixTests : IClassFixture<ResourceApiFixture>, IAsy
         private Guid testResponsibilityMatrixId;
 
         private FusionTestProjectBuilder testProject;
-
-
         private HttpClient client => fixture.ApiFactory.CreateClient();
 
         public ResponsibilityMatrixTests(ResourceApiFixture fixture, ITestOutputHelper output)
```

Comment:
Since you now require unit to be set.
You could maybe prove it by creating a test that fails if you don't set it.